arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -2069,362 +2501,3442 @@
+        "provider": {
+          "description": "The localized friendly form of the resource provider name - it is expected to also include the publisher/company responsible. It should use Title Casing and begin with 'Microsoft' for 1st party services.",
+          "type": "string"
+        },
+        "resource": {
+          "description": "The localized friendly form of the resource type related to this action/operation - it should match the public documentation for the resource provider. It should use Title Casing - for examples, please refer to the 'name' section.",
+          "type": "string"
+        },
+        "operation": {
+          "description": "The localized friendly name for the operation, as it should be shown to the user. It should be concise (to fit in drop downs) but clear (i.e. self-documenting). It should use Title Casing and include the entity/resource to which it applies.",
+          "type": "string"
+        },
+        "description": {
+          "description": "The localized friendly description for the operation, as it should be shown to the user. It should be thorough, yet concise - it will be used in tool tips and detailed views.",
+          "type": "string"
+        }
+      }
+    },
+    "AvailableProviderOperationList": {
+      "description": "List of available provider operations.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The value.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AvailableProviderOperation"
+          }
+        },
+        "nextLink": {
+          "description": "The NextLink.",
+          "type": "string"
+        }
+      }
+    },
+    "AvailableProviderOperationProperties": {
+      "description": "Represents properties of AvailableProviderOperation",
+      "type": "object",
+      "properties": {}
+    },
+    "Backup": {
+      "description": "The backup.",
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseModel"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/BackupProperties",
+          "description": "The properties of the backup.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "BackupElement": {
+      "description": "The backup element.",
+      "required": [
+        "elementId",
+        "elementName",
+        "elementType",
+        "sizeInBytes",
+        "volumeName",
+        "volumeContainerId"
+      ],
+      "type": "object",
+      "properties": {
+        "elementId": {
+          "description": "The path ID that uniquely identifies the backup element.",
+          "type": "string"
+        },
+        "elementName": {
+          "description": "The name of the backup element.",
+          "type": "string"
+        },
+        "elementType
```

Comment:
@ravbhatnagar , @veronicagg @jhendrixMSFT  In all our existing other Services too, we have only been marking "Manager" (the billable entity) for x-ms-azure-resource. Similar in first PR too. How do we go about this here?